src/seo.ts

```diff
@@ -1,16 +1,22 @@
 import { BasePlugin, KalturaPlayer } from '@playkit-js/kaltura-player-js';
 import { convertDurationToISO8601, convertUnixTimestampToISO8601 } from './date-formaters';
 import type { Clip, VideoObject, WithContext } from 'schema-dts';
-import { Chapter, CuePoint, TimedMetadataEvent } from './types';
+import { Chapter, CuePoint, TimedMetadataEvent, UnisphereCuePoint, UnisphereDataEvent } from './types';
 
 export const PLUGIN_NAME = 'seo';
 const SEO_SCRIPT_ID = `${location.hostname}k-player-seo`;
+enum CueSourceNames {
+  None = 'none',
+  TimedMetadata = 'timedMetadata',
+  Unisphere = 'unisphere'
+}
 
 export class Seo extends BasePlugin<Record<string, never>> {
   private chaptersData?: Chapter[];
   private transcriptData?: string;
   private timedDataReadyPromise: Promise<void>;
   private resolveTimedDataReadyPromise!: () => void;
+  private cuesSource: CueSourceNames = CueSourceNames.None;
```

Comment:
I think it's best to add a comment here, something like:
"used to determine where to take chapters data from. unisphere chapters have precedence over cuepoint chapters"